lib/features/offline_mode/observer/sending_email_observer.dart

```diff
@@ -14,15 +11,14 @@ class SendingEmailObserver extends WorkObserver<Email> {
   factory SendingEmailObserver() => _instance ??= SendingEmailObserver._();
 
   @override
-  void handleFailureViewState(Failure failure) {
+  Future<void> observe(String taskId, Map<String, dynamic> inputData) async {
+    log('SendingEmailObserver::observe():taskId: $taskId | inputData: $inputData');
+    return Future.value();
   }
 
   @override
-  void handleSuccessViewState(Success success) {
-  }
-
-  @override
-  WorkStatus observe(Email value) {
-    return WorkStatus.failure;
+  Future<void> bindDI() async {
+    log('SendingEmailObserver::bindDI(): ');
+    return Future.value();
```

Comment:
please make sure not put instance duplicated

Reply:
The task is run in the background. So it's always a unique instance